### PR TITLE
INT-5222 | Decouple IQ Docker release from RedHat registry certifications

### PR DESCRIPTION
**Pull request by fblancosona**
INT-5222 | Decouple IQ Docker release from RedHat registry certifications

- Removed Red Hat Certified Image Build stage from from the main Jenkinsfile
- Created a new Jenkinsfile.rh file to run Jenkins job [ iq-redhat-certified](https://jenkins.ci.sonatype.dev/job/integrations/job/Red%20Hat%20Certified%20Docker%20Image/job/iq-redhat-certified/ )

Links
Jira: https://issues.sonatype.org/browse/INT-5222
Jenkins: https://jenkins.ci.sonatype.dev/job/integrations/job/Red%20Hat%20Certified%20Docker%20Image/job/iq-redhat-certified/

### CODE REVIEW

**Review by fernau — COMMENTED** [review]
this creates a lot of duplication. is there a way to avoid this? 
hmm. it seems the Jira ticket asks for it to be done like this. so I guess that's fine. didn't look at the ticket before.

**Comment by jflinchbaugh** [comment]
nit for next time: branches don't live for all that long, so maybe it doesn't matter much, but i think it's nice to have a few words of description in the branch name in addition to the jira id. `INT-5222-decouple-redhat-from-iq-docker` or something.

File: Jenkinsfile.rh
Comment (jflinchbaugh) [inline Jenkinsfile.rh]: I feel like this build may not need to bother doing any local `docker` stuff, so those calls can likely be removed from this jenkins file.

File: Jenkinsfile.rh
Comment (jflinchbaugh) [inline Jenkinsfile.rh]: is this connection to github statuses otherwise unused?  that may be able to go away.  the main job can update repo status, and this one doesn't need to bother.

File: Jenkinsfile.rh
Comment (jflinchbaugh) [inline Jenkinsfile.rh:28]: this call is the meat of what this job needs to run. anything that's not in direct support of making this call can likely be removed from this jenkinsfile. ✂️

**Comment by jflinchbaugh** [comment]
@fblancosona i've suggested a bunch of things that can be removed, but you'll need to verify that. i can often be wrong. :)

File: Jenkinsfile.rh
Comment (jflinchbaugh) [inline Jenkinsfile.rh:48]: I expect that these 2 utility methods can likely be removed from the old `Jenkinsfile` along with the one usage in Preparation stage. (It's difficult to know what's significant or not the first time you run through these Jenkinsfiles. I have a clue about some of these utility methods, since I'm the one who added them a while ago.)

File: Jenkinsfile.rh
Comment (fblancosona) [reply Jenkinsfile.rh]: @jflinchbaugh It's make sense, I'm going to refactor/ remove everything is not direct related with TriggerRedHatBuild.groovy call

File: Jenkinsfile.rh
Comment (collinpeters) [inline Jenkinsfile.rh]: From what I can tell, the only one of these four that is used is `version`. Can remove the others I think.

File: Jenkinsfile.rh
Comment (collinpeters) [inline Jenkinsfile.rh]: You'll need the `checkout` above to get the `version` below, but I think the rest of this can be removed.

File: ci/TriggerRedHatBuild.groovy
Comment (collinpeters) [inline ci/TriggerRedHatBuild.groovy]: Reminder to uncomment before merge.

**Review by collinpeters — COMMENTED** [review]
Couple of things can still be pruned I think.
@jflinchbaugh should we increase the 20 minute timeout?

File: Jenkinsfile.rh
Comment (fblancosona) [reply Jenkinsfile.rh:48]: @jflinchbaugh in old Jenkinsfile there a couple references to utility methods, so at least for now, I think we can keep it

File: ci/TriggerRedHatBuild.groovy
Comment (fblancosona) [reply ci/TriggerRedHatBuild.groovy]: @collinpeters yes uncommented now

**Comment by jflinchbaugh** [comment]
> Couple of things can still be pruned I think.
> @jflinchbaugh should we increase the 20 minute timeout?

and just accept red hat's atrocious behavior?! :) yeah, since it'll not hold up the other build, a 60-minute timeout seems like it would be fine. ⏱️

**Review by collinpeters — APPROVED** [review]
+1 LGTM

File: Jenkinsfile.rh
Comment (jflinchbaugh) [inline Jenkinsfile.rh]: some of these vars can go away (and the GitHub import), I think.

**Review by jflinchbaugh — APPROVED** [review]
+1 with one extra little cleanup noted. nice work. thanks.

**Pull request merged**
